### ISSUE TITLE
log scale axis for glm-objects (and alike)

### ISSUE THREAD

**Issue by sjPlot** [fsolt/dotwhisker#30]
I'm not sure how this could be addressed the most convenient way, but - for instance - odds ratios should be plotted on a log-scale. I'm not sure if it's possible to see a) whether a tidy data frame from glm-objects has already exponentiated logits (so, odds ratios) and b) whether it's a tidy data frame of a glm-object at all?

**Comment by fsolt** [comment]
Thanks for bringing up odds ratios, Daniel.  In my discipline, political science, at least in the U.S., odds ratios are very much frowned upon (there's a strong pref for predicted probabilities), though I know such norms vary and that odds ratios have their advantages as well.

Hmm, I don't think there's any way for dwplot to know what fxn produced the results in your df.  OTOH, it should be pretty straightforward for get a log scale of one wants one.  Because dwplot outputs a ggplot object, one can just add "+ scale_x_continuous(trans=scales::log_trans())"  

Yes, a quick trial confirms that works fine, but it's much easier to pass confidence intervals than std errors for odds ratios, so dwplot will need to be tweaked to let that happen, and an example added to the vignette won't hurt either.

![log_scale](https://cloud.githubusercontent.com/assets/6536645/9034971/6466452c-3999-11e5-9386-0466188a1c2a.png)
